### DISCUSSION TITLE
Is there a way to specify download site for all mods?

### DISCUSSION THREAD

**Discussion by danielpza** [Senth/minecraft-mod-manager#130] [Q&A]
it can be tedious to specify the download site for each mod manually

**Comment by Senth** [comment]
Hi Daniel 🙂 

In general, you don't have to specify the download site for each mod. By default, `mcman` will try to find the mod on both CurseForge and Modrinth and download the newest. But if you just want to download mods from one site there's, unfortunately, no way to do that easily.

There might be an addition after #38 has been implemented.
